src/components/ListSearchResult.js

```diff
@@ -77,22 +82,8 @@ class InnerListSearchResult extends React.Component<InnerListSearchResultDefault
   /**
    * Renders a <ListSearchResult> component for the document.
    */
-  static renderer(
-    doc: SearchDocument,
-    position: number,
-    baseUri: string,
-    key: string,
-    hide360Link: boolean = false,
-  ) {
-    return (
-      <ListSearchResult
-        document={doc}
-        position={position}
-        baseUri={baseUri}
-        key={key}
-        hide360Link={hide360Link}
-      />
-    );
+  static renderer(doc: SearchDocument, position: number, baseUri: string, key: string, hide360Link: boolean = false) {
+    return <ListSearchResult document={doc} position={position} baseUri={baseUri} key={key} hide360Link={hide360Link} />;
```

Comment:
Prettier? Let's keep this as it was.
```suggestion
static renderer(
  doc: SearchDocument,
  position: number,
  baseUri: string,
  key: string,
  hide360Link: boolean = false,
  ) {
    return (
      <ListSearchResult
        document={doc}
        position={position}
        baseUri={baseUri}
        key={key}
        hide360Link={hide360Link}
      />
    );
```